receiver/hostmetricsreceiver/hostmetrics_linux.go

```diff
@@ -0,0 +1,58 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build linux
+
+package hostmetricsreceiver
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+func validateRootPath(rootPath string) error {
+	if rootPath == "" {
```

Comment:
Should return if it's "" or "/"?

Reply:
I thought a better default for `RootPath` is `""`, because `/` is only the rootpath on unix environments. But I do agree, probably fine to return here if the user has explicitly set it as `/`. I will add it in.